### DISCUSSION TITLE
Discord account got hacked

### DISCUSSION THREAD

**Discussion by beef-techau** [RavenSystem/esp-homekit-devices#2155] [General]
Hi José, my discord got hacked and I got banned from the discord channel. I've changed password and set up 2FA, I even created a new discord account and I can't rejoin the discord. Please 🙏🥹

**Comment by RavenSystem** [comment]
Sure. Unbanned. Try rejoin.

**Reply by beef-techau** [reply]
🙏 thank you. My apologies how frustrating. I've got to try and do this to all the servers I was in. 
I even created new account and still didn't work.

**Reply by RavenSystem** [reply]
Don't worry. 😉
